src/org/omegat/gui/glossary/GlossaryTextArea.java

```diff
@@ -167,10 +167,30 @@ public Component getComponentToOverlay() {
         }
 
         ToolTipManager.sharedInstance().registerComponent(this);
+        try {
+            String preference = Preferences.getPreferenceDefault(Preferences.GLOSSARY_LAYOUT,
+                entryRenderer.getId());
+            for (IGlossaryRenderer r: Core.getGlossaryRenderers()) {
+                if (r.getId().equals(preference)) {
+                    entryRenderer = r;
+                }
+            }
+        } catch (Exception e) {
+        }
```

Comment:
What is the exception that could be thrown here?

Reply:
You are true, I just removed it.